pages/index.vue

```diff
@@ -50,34 +39,50 @@
         preview: query.mode === 'preview'
       };
 
-      return app.$contentful.query('browsePage', variables)
+      return app.$contentful.query('browseStaticPage', variables)
         .then(response => response.data.data)
         .then(data => {
-          if (data.browsePageCollection.items.length === 0) {
+          if (data.staticPageCollection.items.length > 0) {
+            let itemData = data.staticPageCollection.items[0];
+            itemData.staticPage = true;
+            return itemData;
+          } else if (data.browsePageCollection.items.length > 0) {
+            let itemData = data.browsePageCollection.items[0];
+            itemData.browsePage = true;
+            return itemData;
+          } else if (data.browsePageCollection.items.length === 0) {
```

Comment:
no need for the if  condition, just have it be `else {` or use a default return without any condition as was happening before, just with a 404 as the default.